src/storage/index/LookUpIndexBaseProcessor.inl

```diff
@@ -0,0 +1,201 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include <interface/gen-cpp2/storage_types.h>
+#include <interface/gen-cpp2/common_types.h>
+
+DECLARE_int32(max_row_returned_per_index_scan);
+DECLARE_bool(enable_vertex_cache);
+
+namespace nebula {
+namespace storage {
+template<typename REQ, typename RESP>
+cpp2::ErrorCode ScanIndexBaseProcessor<REQ, RESP>::buildIndexHint() {
+    // TODO(sky) : Currently only supported :
+    //             1) Range query for the first column
+    //             2) Equivalent query for all columns
+    if (hint_.is_range) {
+        if (hint_.hint_items.size() != 1) {
+            return cpp2::ErrorCode::E_INVALID_INDEX_HINT;
+        }
+        range_ = std::make_pair(hint_.hint_items[0].get_first_str(),
+                                hint_.hint_items[0].get_second_str());
+    } else {
+        prefix_.reserve(128);
+        for (auto& hint : hint_.hint_items) {
+                prefix_.append(hint.get_first_str());
```

Comment:
Who take care of the index hint order? I guess graph check the what kind of index we have, and if thery are the first N columns?